docs/book/releases/v2.1.0.md

```diff
@@ -30,10 +30,10 @@ Note: For vSphere CSI Migration feature the minimum Kubernetes version requireme
 
 1. When the static persistent volume with file-share is re-created with the same PV name, volume is not getting registered as a container volume with vSphere.
    - Impact: attach/delete can not be performed on the such Persistent Volume.
-   - Workaround: wait for 1 hour before re-creating static persistent volume with file-share using the same name.
+   - Workaround: wait for 1 hour before re-creating static persistent volume with file-share using the same name. Note: This issue is fixed in [v2.2.0](v2.2.0.md)
 2. Metadata syncer container deletes the volume physically from the datastore when Persistent Volumes with `Bound` status and reclaim policy `Delete` is deleted by the user when `StorageObjectInUseProtection` is disabled on Kubernetes Cluster.
    - Impact: Persistent Volumes Claim goes in the lost status. Volume can not be recovered.
-   - Workaround: Do not disable `StorageObjectInUseProtection` and attempt to delete Persistent Volume directly without deleting PVC.
+   - Workaround: Do not disable `StorageObjectInUseProtection` and attempt to delete Persistent Volume directly without deleting PVC. Note: This issue is fixed in [v2.2.0](v2.2.0.md)
```

Comment:
same here

Reply:
just a note would be good enough to understand that upgrading the driver to v2.2.0 will fix this issue.